xbmc/windowing/android/AndroidUtils.h

```diff
@@ -31,7 +31,21 @@ class CAndroidUtils : public ISettingCallback
   static const std::string SETTING_LIMITGUI;
   void OnSettingChanged(const std::shared_ptr<const CSetting>& setting) override;
 
+  // HDR info
+  enum HDRTypes
+  {
+    DOLBY_VISION = 1,
+    HDR10 = 2,
+    HLG = 3,
+    HDR10_PLUS = 4
+  };
```

Comment:
in an enum is not needed assign the value on each item only in the first one to set the starting value, unless you intend to do something specific in the future

Reply:
now understand, this is a mapping of https://developer.android.com/reference/android/view/Display.HdrCapabilities#constants_1
then maybe here is better add a comment on this enum that this refer to these android constants?

Reply:
I thought the fact that it's in AndroidUtils was implying that, but I'll add a comment.